### ISSUE TITLE
Support custom fonts

### ISSUE THREAD

**Issue by GrahLnn** [marimo-team/marimo#622]
### Description

Will Marimo support custom font configuration for cell display? It's a bit regrettable that such a beautiful notebook cannot have custom fonts.

### Suggested solution

Can choosing code fonts and ligatures, such as Fira Code, selection different languages fonts (mainly non-English text).

**Comment by mscolnick** [comment]
While the css variables we use are not quite a supported API - you can override this if you'd like. 

After [this PR is released](https://github.com/marimo-team/marimo/pull/630), you can do something like:

```python
import marimo as mo

# Load the font and override the css vars
mo.Html(
    """
<link href="https://fonts.googleapis.com/css?family=Happy+Monkey" rel="stylesheet">

<style>
    :root {
        --text-font: "Happy Monkey", cursive, sans-serif;
        --heading-font: "Happy Monkey", cursive, sans-serif;
    }
</style>
"""
)
```

<img width="900" alt="Screenshot 2024-01-22 at 11 12 34 PM" src="https://github.com/marimo-team/marimo/assets/2753772/46555cba-ca85-4a0f-b704-a35299d88f93">


If this fits your needs, happy to support this. Otherwise open to discuss further font customization

**Comment by akshayka** [comment]
@GrahLnn , we just released 0.1.81, which lets you do what Myles suggested in the previous post. Let us know if that works for you.
